src/components/modes/teacher/ImageUpload.tsx

```diff
@@ -120,6 +124,12 @@ const ImageUpload = (): JSX.Element => {
         <Typography sx={{ color: '#383838' }}>
           {t(`Max file size: ${MAX_FILE_SIZE / 1e6}mb`)}
         </Typography>
+        <Typography sx={{ color: '#383838' }}>
```

Comment:
You can factor your typography out into a custom styled component:
```ts
const StyledText = styled(Typography)({
  color: '#383838',
});
``` 
And then use it here and above.